simple_parsing/wrappers/field_wrapper.py

```diff
@@ -19,6 +19,19 @@
 logger = getLogger(__name__)
 
 
+class FullPathMode(Enum):
+    """
+    Whether or not to force the full path to be used for the option string.
+    When set to FullPathMode.DISABLED (default), the option string is as
+    flatten as possible. When set to FullPathMode.FULL, the option string
+    is the full path to the field. Sometimes is desired to ignore the
+    first level. Use FullPathMode.FULL_WITHOUT_ROOT in such cases.
```

Comment:
Would you mind re-organizing this a little bit into more of a bullet-point list for each option, and using the same example argument to illustrate each option? Like "--baz", "--foo.bar.baz", "--bar.baz"?

Alternatively, you could add a "docstring" under each enum value with a description.